tests/std/tests/P0896R4_views_common/test.cpp

```diff
@@ -387,8 +387,128 @@ struct instantiator {
     }
 };
 
+template <class T>
+struct difference_type_only_iterator {
+    static_assert(is_object_v<T>);
+
+    friend constexpr bool operator==(difference_type_only_iterator, difference_type_only_iterator)  = default;
+    friend constexpr auto operator<=>(difference_type_only_iterator, difference_type_only_iterator) = default;

```

Comment:
No change requested: This is correct, but defaulting just the spaceship would be sufficient. This is WG21-N4917 [[class.compare.default]/4](https://eel.is/c++draft/class.compare.default#4).